tests/cluster/mount/__init__.py

```diff
@@ -1,6 +1,34 @@
-"""
-This folder does not contain any unittest code.
+from atc.config_master import TableConfigurator
+from atc.functions import init_dbutils
+from atc.spark import Spark
 
-The json file in this folder is passed as an argument to the setup job
-that configures the test environment. Therefore, the mount code is tested.
-"""
+
+def prepare_environment():
+    # with the following lines, the storage account can be mounted directly
+
+    scope = "secrets"
+    dbutils = init_dbutils()
+    spark = Spark.get()
+
+    clientSecret = dbutils.secrets.get(scope=scope, key="Databricks--ClientSecret")
+    clientId = dbutils.secrets.get(scope=scope, key="Databricks--ClientId")
+    tenant = dbutils.secrets.get(scope=scope, key="Databricks--TenantId")
+    storageAccount = "githubatc.dfs.core.windows.net"
+
+    prfx = "fs.azure.account"
+    spark.conf.set(f"{prfx}.auth.type.{storageAccount}", "OAuth")
+    spark.conf.set(
+        f"{prfx}.oauth.provider.type.{storageAccount}",
+        "org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",
+    )
+    spark.conf.set(f"{prfx}.oauth2.client.id.{storageAccount}", clientId)
+    spark.conf.set(f"{prfx}.oauth2.client.secret.{storageAccount}", clientSecret)
+    spark.conf.set(
+        f"{prfx}.oauth2.client.endpoint.{storageAccount}",
+        f"https://login.microsoftonline.com/{tenant}/oauth2/token",
+    )
+
+    # This is an abbreviation to access the storage account
+    TableConfigurator().register(
+        "storageAccount", dict(name=f"abfss://silver@{storageAccount}")
```

Comment:
if the function should be more general available, this part  (line 32-33) should might be rethinked. Is this general no matter the dataplatform?

If not, we could make the function more general minus line 32-33. Then in make a new function called ```prepare_atc_environment()``` calls the general "prepare_environmen(*inputs...*)" and then calls line 32-33

Reply:
maybe in a new PR. I'm not sure how to generalize this since there are other ways of authenticating.